src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -37,18 +37,29 @@ const char* NPU_PLUGIN_LIB_NAME = "openvino_intel_npu_plugin";
  * @param resultDescriptors Describes the output nodes.
  * @param inputNames The names of the inputs registered in the order given by the model.
  * @param outputNames The names of the outputs registered in the order given by the model.
+ * @param config
  */
 std::shared_ptr<ov::Model> create_dummy_model(const IONodeDescriptorMap& parameterDescriptors,
                                               const IONodeDescriptorMap& resultDescriptors,
                                               const std::vector<std::string>& inputNames,
-                                              const std::vector<std::string>& outputNames) {
+                                              const std::vector<std::string>& outputNames,
+                                              std::shared_ptr<NPUBackends> backends) {
```

Comment:
I see that backends is used only for extracting a boolean flag. Can you consider passing the flat as an arg instead then?

Reply:
done, thanks!